.gitignore

```diff
@@ -16,3 +16,4 @@ tests/cypress/downloads
 .vscode
 /.cdk
 cdk.context.json
+**/.yarn/*
```

Comment:
Interesting!
How does it use this folder? Does it do this instead of node_modules or is it being used for another purpose?

Reply:
it uses it in addition to node_modules